### DISCUSSION TITLE
No menu bar on version 3.1.3

### DISCUSSION THREAD

**Discussion by dl98709z9** [audacity/audacity#3502] [General]
Got a Mac; don't know why the version I download from audacityteam.org is missing the usually menu bar; watching Josh Meyer on YouTube and hence saw the missing bar;  File, edit, etc.
Have downloaded it several times;
suggestions on How to proceed would be appreciated;

**Comment by LWinterberg** [comment]
on mac, the menu bar typically is at the very top of the screen

**Reply by petersampsonaudacity** [reply]
> on mac, the menu bar typically is at the very top of the screen

But only when Audacity is the active app ...

**Comment by dl98709z9** [comment]
Thanks; I got it working now;


On Wed, Sep 7, 2022 at 5:27 AM Peter Sampson ***@***.***>
wrote:

> on mac, the menu bar typically is at the very top of the screen
>
> But only when Audacity is the active app ...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/audacity/audacity/discussions/3502#discussioncomment-3583150>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKPV3MXGRI32WAGIR7D2HWDV5BNXNANCNFSM6AAAAAAQBUO3NI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Denis Silverman
